### ISSUE TITLE
Upload latest aarch64 packages 

### ISSUE THREAD

**Issue by andfoy** [conda-forge/qt-main-feedstock#15]
@conda-forge/core @hmaarrfk 

I've just built and uploaded to my channel the latest build of `qt-main` for `aarch64`. https://anaconda.org/andfoy/qt-main/files

Here is the corresponding log: https://drive.google.com/file/d/1bm9l7IM6G8FL32trk-d_iufYpUGDdluP/view?usp=sharing

I would appreciate your help with this in order to move https://github.com/conda-forge/qt-webengine-feedstock/issues/2 forward.

**Comment by ngam** [comment]
@andfoy, sorry for your trouble doing this (I am not part of core; I have no powers around here). As many would tell you, this manual upload business is a relatively burdensome process. It also happens in the tensorflow and pytorch feedstocks. I thought about this a while ago, for example, and I was surprised that @hmaarrfk was not a core member --- surprising given the solid and long track of ensuring tensorflow and pytorch are up to date (he is now, fully deservedly). 

I think generally speaking, something to advocate for is increasing the number of core members or maybe creating a team with lower credentials who can do some of what core members can do (e.g. merging random PRs all over the place especially in abandoned feedstocks, manually uploading some packages, etc.). The motivation is that, core members --- like other members of the community --- are volunteers and thus their schedules may vary from month to month and this dynamic creates inconsistencies and may frustrate contributors like yourself :( 

To start (from my limited experience around the tensorflow/pytorch feedstocks), I would personally nominate @h-vetinari and @Tobias-Fischer and @izahn to be part of core or whatever the team just below core might be (if they so consent, of course, just picking people I notice around who are helpful). I am not sure if there is a formal process to nominate people, maybe there should be... Anyway, I think that would go a long way to reduce the burden and making the process more efficient (and arguably more equitable, etc. but that's besides the point for now).

Just my 2c, cc @conda-forge/core

**Comment by ngam** [comment]
(It's also perhaps necessary to have like an activity measure --- some core members may decide to move on or become inactive for whatever reason; in that case, they should no longer be part of core. Some volunteer organizations have quotas like being present for 75% of meetings, doing xyz number of tasks per month, etc. to keep your position as part of the leading/admin committee and so this may apply here... but then again, I am new to all of this so idk if such a thing exists anyway)

**Comment by beckermr** [comment]
Please see our documentation for current policies around the core team, activity, etc. We welcome new nominations to core and take our collective responsibility very seriously. 

What I think is important to remember about the specific motivation here is that this discussion is about uploading artifacts built outside of the open CI systems we normally use. That process is **purposefully gummed up** because of the extra degree of trust it requires. 

The core team is hard at work obtaining CI resources that can run beyond the limits imposed by azure and with GPUs (!!!). These resources, once they are fully provisioned, will ease these issues and improve the overall quality of our heavy-to-build packages beyond the improvement of more timely uploads. We will be able to run smoke tests on GPU hardware with them and be able to build them in the open so that everyone can see exactly how they were built, with what software, know the machine used to build them had some degree of being properly secured, was wiped properly between builds, etc.

**Comment by isuruf** [comment]
@ngam, please stay on topic. If you want to open a different issue.

**Comment by ngam** [comment]
@beckermr yes I agree with all you said, and sorry @isuruf I thought this was _maybe_ relevant here. My bad!

**Comment by hmaarrfk** [comment]
@andfoy I can upload them, but if you could add a label to the package, then i can more readily follow my instructions from here
https://github.com/conda-forge/cfep/blob/main/cfep-03.md

The less I have to think, the faster I can work!

Thanks!

**Comment by hmaarrfk** [comment]
Sorry for the extra trouble, but thakn you for taking the time to compile!

**Comment by hmaarrfk** [comment]
42MB log: [linux-aarch64-build.zip](https://github.com/conda-forge/qt-main-feedstock/files/8182362/linux-aarch64-build.zip)

I just uploaded the package. Found out the right command.

Thanks for your work!

**Comment by hmaarrfk** [comment]
For my own reference:
```
anaconda copy  --to-label main --to-owner conda-forge andfoy/qt-main/5.15.2/linux-aarch64/qt-main-5.15.2-h7c38af3_3.tar.bz2
```

I may ask in the future for a label, especially when it becomes more than 1 package.
